src/plonk/mod.rs

```diff
@@ -59,15 +62,16 @@ pub struct PlonkStructure<C: CurveAffine> {
 pub struct PlonkInstance<C: CurveAffine> {
     pub(crate) W_commitment: C, // concatenate num_advice_columns together, then commit
     pub(crate) instance: Vec<C::ScalarExt>, // inst = [X0, X1]
-    pub(crate) y: C::ScalarExt,
+    // contains challenges
```

Comment:
Explain a little what is challenge. Not very deeply, but give general ctx
And from comment to rustdoc `//` -> `///`